tests/utils.go

```diff
@@ -322,6 +326,8 @@ func AfterTestSuitCleanup() {
 	DeletePVC(osAlpineHostPath)
 	DeletePV(osAlpineHostPath)
 
+	cleanupHostDiskData()
```

Comment:
The last thing I saw: The pod created for deleting this is cleaning up asynchronously. That implies a few things:

1) If this line succeeds, the data from the node is not really deleted
2) we would create a lot of cleanup pods

I think that you need to run this cleanup especially for the related tests (before and afterwards) and also wait for the pod to succeed or fail.

Reply:
Yes you're right, as well I have to handle a situation where there is more than one node, basically, I think that's why my tests are failing on ```check-patch.k8s-1.11.0-dev.el7.x86_64 ```, I am just lucky that others pass.
In progress...